src/workerd/api/node/buffer.h

```diff
@@ -77,6 +77,8 @@ class BufferUtil final: public jsg::Object {
                         kj::Array<kj::byte> bytes,
                         kj::Array<kj::byte> state);
   jsg::JsString flush(jsg::Lock& js, kj::Array<kj::byte> state);
+  bool isAscii(kj::Array<kj::byte> bytes);
```

Comment:
does this mean that the argument can be any of `<Buffer> | <ArrayBuffer> | <TypedArray> ` as in node? https://nodejs.org/api/buffer.html#buffer_buffer_isascii_input

Reply:
yes. the one difference is that I'm not checking to see if the underlying ArrayBuffer has been transfered

Reply:
Is that intentional?

Reply:
For now, yes, since we have significantly less use case for transfered buffers since we do not have worker threads. We do have `structuredClone(...)` and `ArrayBuffer`s can be transfered with streams but it's likely quite unlikely for it to be an issue.

Reply:
could you lave a note in the code as a comment? this kind of context is not obvious and will become handy down the road. thank you